TODO.md

```diff
@@ -7,6 +7,18 @@ Scratchpad for random ideas of things that need done.
 * Verify RDP enable
 * Linux repos
 * Whole buncha stuff for profile::cfcc::camper
+Charlie's $0.02
+
+* jdk8 > jdk10
+* remove eclipse
+* add a good text editor (I use notepad2-mod; I think lots of the others are overkill and overwhelming, nbd though)
+* Add registry entry to allow edit of any file from context menu in classesroot/*/editinnotepad[Edit in Notepad]/command/notepad.exe %1
```

Comment:
:+1:

Reply:
@idontusenumbers Do you have a sample reg key for that "edit in notepad" feature?